packages/services/schema/src/lib/__tests__/federation-tag-extraction.spec.ts

```diff
@@ -1813,7 +1974,7 @@ describe('applyTagFilterOnSubgraphs', () => {
 
     expect(print(result[0].typeDefs)).toMatchInlineSnapshot(`
       type Query {
-        field1: String! @inaccessible
+        field1: String!
```

Comment:
Since we now create type register before, this is no longer marked as inaccessible.